### PR TITLE
Bonnie 1513

### PR DESCRIPTION
**Pull request by mayerm94**
- [x]  ❗️ Must have HDS Gemfile reference updated after associated PR is merged  https://github.com/projectcypress/health-data-standards/pull/578
- [x]  ❗️ Must have cql_qdm_patientapi Gemfile reference updated after associated PR is merged. https://github.com/projecttacoma/cql_qdm_patientapi/pull/133

Add method and qdm_status to datatypes that required them, but did not have them
See model info file and QDM spec for proof that these attributes are required on the modified datatypes https://github.com/cqframework/clinical_quality_language/blob/master/Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1513
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. NA
- [x] Tests are included and test edge cases NA
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 

Branch | Back End Coverage | Front End Coverage
-- | -- | --
master | [1738 / 1950 LOC (89.13%)] | [Statements   : 58.45% ( 7037/12040 ) Branches     : 42.21% ( 1888/4473 ) Functions    : 48.85% ( 1167/2389 ) Lines        : 56.27% ( 6311/11215 )]
this branch | [1738 / 1950 LOC (89.13%] |  [Statements   : 58.45% ( 7037/12040 ) Branches     : 42.21% ( 1888/4473 ) Functions    : 48.85% ( 1167/2389 ) Lines        : 56.27% ( 6311/11215 )]

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: https://jira.mitre.org/browse/BONNIE-1524
- [x] Justification for using JIRA tests: Round-trip of data (creation->calculation/storage)
- [x] JIRA tests have been added to sprint 

**Reviewer 1:**

Name: @c-monkey
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Review by c-monkey — DISMISSED** [review]
Codes appear as appropriate and this PR works with the other 2 Bonnie 1513 PRs (cql_qdm_patientapi and hds)

File: Gemfile [hunk 1]
```diff
@@ -13,15 +13,15 @@ gem 'less-rails'
...
+# gem 'health-data-standards', '~> 4.0.5'
 gem 'cql_qdm_patientapi', '~> 1.1.3'
 gem 'simplexml_parser', '~> 1.0'
 gem 'hqmf2js', '~> 1.4'
 gem 'bonnie_bundler', '~> 2.1.1'
 gem 'quality-measure-engine', '~> 3.2'
 gem 'hquery-patient-api', '~> 1.1'
 
-# gem 'health-data-standards', :git => 'https://github.com/projectcypress/health-data-standards.git', :branch => 'master_bonnie'
+  gem 'health-data-standards', :git => 'https://github.com/projectcypress/health-data-standards.git', :branch => 'master_bonnie'
```
Comment (hossenlopp) [inline Gemfile]: Should not have any spaces before `gem`.

File: Gemfile.lock [hunk 2]
```diff
@@ -377,7 +383,7 @@ GEM
       unf_ext
     unf_ext (0.0.7.5)
     unicode-display_width (1.3.0)
-    uuid (2.3.8)
+    uuid (2.3.9)
```
Comment (hossenlopp) [inline Gemfile.lock]: This shouldn't have been updated.

**Review by hossenlopp — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (mayerm94) [reply Gemfile]: I know --intend on fixing when the cql_qdm_patientapi hash is updated once the 1513 PR is merged in that repo

File: Gemfile.lock [hunk 3]
```diff
@@ -383,7 +389,7 @@ GEM
       unf_ext
     unf_ext (0.0.7.5)
     unicode-display_width (1.3.0)
-    uuid (2.3.9)
+    uuid (2.3.8)
```
Comment (hossenlopp) [inline Gemfile.lock]: This line change should be reverted.

**Review by hossenlopp — DISMISSED** [review]
JIRA test pass. Looks good.

**Review by jbradl11 — DISMISSED** [review]
Looks good. Nice Jira Test!

**Review by hossenlopp — APPROVED** [review]

**Review by jbradl11 — APPROVED** [review]
Changes still look good.

**Pull request merged**